FEXCore/Source/Interface/Core/OpcodeDispatcher.h

```diff
@@ -1198,6 +1209,10 @@ class OpDispatchBuilder final : public IREmitter {
 
   OrderedNode* GetRelocatedPC(const FEXCore::X86Tables::DecodedOp& Op, int64_t Offset = 0);
 
+  struct AddressMode AddSegmentToAddress(struct AddressMode A, uint32_t Flags);
```

Comment:
Struct types don't need to be prefixed with `struct ` in C++.

(I'm actually surprised clang-format doesn't format it away, since it really looks out of place.)